packages/apollo/src/Command.ts

```diff
@@ -103,6 +103,13 @@ export abstract class ProjectCommand extends Command {
     this.ctx = { flags, args } as any;
 
     const config = await this.createConfig(flags);
+
+    if (!config) {
```

Comment:
You've already handled the null config case within `this.createConfig()`, I think this is redundant.

Reply:
We handled the case of not having a config, but the command itself still needs to exit. It can't call `this.createService` with an empty config.